libraries/common/effekt.effekt

```diff
@@ -113,7 +113,7 @@ extern io def inspect[R](value: R): Unit =
 // Strings
 // =======
 extern pure def infixConcat(s1: String, s2: String): String =
-  js "${s1} + ${s2}"
+  js "((${s1}) + (${s2}))"
```

Comment:
We could also perhaps use `((${s1}).concat(${s2}))` instead, which might be a bit safer (but I haven't tested it thoroughly)

Reply:
"Safer" in which sense?

Reply:
To be honest, I had hoped that it would be more strict [wrt types] than it really is, but it's still "type-safe" in its first argument:
```js
> "abc".concat(5)
"abc5"
> (5).concat("hello")
Uncaught TypeError: 5.concat is not a function
```